compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmLower.kt

```diff
@@ -343,6 +343,7 @@ private val jvmFilePhases = listOf(
     interfaceSuperCallsPhase,
     interfaceDefaultCallsPhase,
     interfaceObjectCallsPhase,
+    jvmStaticAnnotationPhase,
```

Comment:
Can we add some prerequisites on this phase, if this move is necessary?

Reply:
It's not strictly necessary, this is just to ignore `$$forInline` versions of suspend functions created by AddContinuationLowering below since creating static versions of those is obviously pointless.